aslam_cv_calibration/include/aslam/calibration/target-base.h

```diff
@@ -45,11 +45,16 @@ class TargetBase {
   size_t gridCoordinatesToPoint(size_t row_idx, size_t col_idx) const;
 
   /// Number of rows in the calibration target.
-  size_t rows() const { return rows_; };
+  inline size_t rows() const { return rows_; };
   /// Number of columns in the calibration target.
-  size_t cols() const { return cols_; };
+  inline size_t cols() const { return cols_; };
   /// Get the number of points of the full grid.
-  size_t size() const { return rows_ * cols_; };
+  inline size_t size() const { return rows_ * cols_; };
+
+  /// Extent of the grid in row dimension (x by default).
```

Comment:
is x a placeholder?

Reply:
No it should actually denote the x coordinate :-)